plugins/nextcloud/NextcloudAddressBook.php

```diff
@@ -1,90 +1,129 @@
 <?php
+
 use RainLoop\Providers\AddressBook\Classes\Contact;
 
 class NextcloudAddressBook implements \RainLoop\Providers\AddressBook\AddressBookInterface
 {
 	use RainLoop\Providers\AddressBook\CardDAV;
-	private const URI = 'webmail';
 	private $contactsManager;
-	private $key;
 
-	function __construct()
+	private const SETTINGS_KEY = 'nextCloudAddressBookUri';
```

Comment:
Are we following same convention as the other settings?

Also nextcloud is one word, doesn't need to be `camelCased`

Reply:
@akhil1508 please check again